src/main/java/com/opencart/driver/DriverRepository.java

```diff
@@ -18,6 +20,7 @@ public static void downloadWebDriver() {
 
     public static void instanceWebBrowser() {
         webDriver = new ChromeDriver();
+        webDriver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
```

Comment:
Don't use implicitWaits - it is bad practice, remove it, and use explicit ones when necessary